packages/connector-local/src/connector.ts

```diff
@@ -97,7 +98,7 @@ export default class LocalConnector implements IConnector {
          */
         const uri: url.URL = getAsUri(target);
         const filePath: string = asPathString(uri);
-        const content: NetworkData = await this.fetchContent(filePath);
+        const content: NetworkData = await this.fetchContent(filePath, options);
```

Comment:
`await this.fetchContent(filePath, null, options);`

otherwise, when you check in line # 253 `options.content` is going to be always null.

Reply:
Good catch, thanks!